.github/workflows/_build_artifacts.yml

```diff
@@ -237,8 +237,10 @@ jobs:
             --clobber \
             --repo ${{ github.repository }}
       - name: Set up QEMU
+        if: ${{ inputs.stage == 'release' || !contains(matrix.arch.shortname, 'arm') }}
```

Comment:
```suggestion
        # Only proceed to build debug / testing images for linux/amd64, otherwise this
        # will hang when trying to `npm i` when the container platform and host platform differ.
        # See https://github.com/nodejs/docker-node/issues/1946
        if: ${{ inputs.stage == 'release' || matrix.arch.platform == 'linux/amd64' }}
```